src/content/1.7/modules/sample-modules/grid-and-identifiable-object-form-hooks-usage.md

```diff
@@ -314,3 +314,144 @@ private function updateCustomerReviewStatus(array $params)
 when we created the switch type form we named it `is_allowed_for_review`. By using the same name we can get the state (on or off).
 This hook receives from `$params` the form data, that you can retrieve like this: `$params['form_data']`.
 All the form data is available here, including `is_allowed_for_review` data which comes from the switch.
+
+The whole main class could look similar to: 
```

Comment:
Is it the right place to put this statement ? The text before seems unrelated to your added statement. Maybe we need a transition between them ?

Reply:
Maybe it's better to provide a link to example module instead of putting whole main class here?

Reply:
it's not the right place to put this statement